modules/layers/src/bitmap-layer/bitmap-layer.js

```diff
@@ -192,7 +198,12 @@ export default class BitmapLayer extends Layer {
 
     if (typeof image === 'string') {
       loadImage(image).then(data => {
-        this.setState({bitmapTexture: new Texture2D(gl, {data})});
+        this.setState({
+          bitmapTexture: new Texture2D(gl, {
+            data,
+            parameters: DEFAULT_TEXTURE_PARAMETERS
```

Comment:
Does this help the image quality?

Reply:
Not really. It looks the same with or without these parameters.